File: src/main/java/ro/sopa/statistifier/web/model/ArtistTimeline.java
package ro.sopa.statistifier.web.model;

import java.util.List;

public class ArtistTimeline {

    private String artist;

    private List<TimelinePoint> points;

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public List<TimelinePoint> getPoints() {
        return points;
    }

    public void setPoints(List<TimelinePoint> points) {
        this.points = points;
    }
}

File: src/main/kotlin/ro/sopa/lastfm/web/model/ArtistTimeline.kt
package ro.sopa.lastfm.web.model

data class ArtistsTimeline(val artistTimelines: List<ArtistTimeline>)

data class ArtistTimeline(val artist: String, val points: List<TimelinePoint>)

data class TimelinePoint (val time: String, val listens: Long)
